arcgis-ios-sdk-samples/Route & Directions/Offline routing/README.md

```diff
@@ -1,20 +1,43 @@
 # Offline routing
 
-This sample demonstrates how to route using a local San Diego streets network dataset loaded into the app.
+Solve a route on-the-fly using offline data.
 
+![Fastest route](offline-routing-1.png)
+![Shortest route](offline-routing-2.png)
+
+## Use case
+
+You can use an offline network to enable routing in disconnected scenarios. For example, you could provide offline location capabilities to field workers repairing critical infrastructure in a disaster when network availability is limited.
 
 ## How to use the sample
 
-You tap on the map to add a stop. As you add individual stops the sample solves the route between the stops. If you tap, hold and move then the route will be calculated dynamically; on the fly. You can also switch between the fastest or the shortest route.
+Tap near a road to start adding a stop to the route, tap again to place it on the map. A number graphic will show its order in the route. After adding at least 2 stops, a route will display. Choose "Fastest" or "Shortest" to control how the route is optimized. The route will update on-the-fly while adding stops. 
+
+## How it works
 
+1. Create the map's `AGSBasemap` from a local tile package using an `AGSTileCache` and `AGSArcGISTiledLayer`.
+2. Create an `AGSRouteTask` with an offline locator geodatabase.
+3. Get the `AGSRouteParameters` using `_.getDefaultParameters()`.
+4. Create `AGSStop`s and add them to the route task's parameters.
+5. Solve the route using `routeTask.solveRoute(with:completion:)`.
```

Comment:
```suggestion
5. Solve the route using `AGSRouteTask.solveRoute(with:completion:)`.
```
Not sure if you know it, but Philip taught me a very useful shortcut to copy the qualified symbol name for the iOS SDK.

Press <kbd>Control</kbd>+<kbd>Command</kbd>+<kbd>Shift</kbd>+<kbd>Option</kbd>+<kbd>C</kbd> together when a symbol is selected, and it will copy the full qualified symbol name for it! 😏

Reply:
Oh, I didn't know that. Thanks! Very helpful 🙂